src/main/steps/common/alldocuments/template.njk

```diff
@@ -0,0 +1,34 @@
+{% extends "common/template.njk" %}
```

Comment:
where this new template is used?

Reply:
Its been used in all the documents templates. for both applicant and respondent.